src/main/java/uk/gov/hmcts/reform/profilesync/service/ProfileSyncService.java

```diff
@@ -7,9 +7,6 @@
 
 public interface ProfileSyncService {
 
-    String BASIC = "Basic ";
-    String BEARER = "Bearer ";
```

Comment:
Sonar: "Move constants to a class or enum"
https://sonarcloud.io/project/issues?id=RDPROFILESYNCAPI&open=AWxuE_h-_Xcr_PhAhRjO&resolved=false

So this was moved to the ServiceImpl (and ServiceImplTest classes too).